ffmuc-mesh-vpn-wireguard-vxlan/files/lib/gluon/gluon-mesh-wireguard-vxlan/checkuplink

```diff
@@ -329,3 +314,12 @@ ip link set up dev mesh-vpn
 sleep 5
 # If we have a BATMAN_V env we need to correct the throughput value now
 batctl hardif mesh-vpn throughput_override 1000mbit;
+
+# Check again if connected
+if is_connected; then
+	logger -p info -t checkuplink "Successfully connected to $PEER_HOST($PEER_ENDPOINT)"
+	exit 0
```

Comment:
I think this exit here is not necessary anymore, right? We only need to make sure we check for `! is_connected()` and then `exit 4`, but not the other way round?

Reply:
do we even catch the exit code somewhere else?
i think a log info that we are connected Successfully is nice tbh. but `exit 0` we can have but we dont need to

Reply:
Yeah, reducing nesting here is probably a good idea 🙂

Reply:
https://github.com/freifunk-gluon/community-packages/pull/106/commits/6db93e417426db260ce87997bee9597110b2d72a removes the nesting